tests/test_cli.py

```diff
@@ -79,8 +79,8 @@ def get_test_options():
         OptionStruct("int", "profile", "--perf-analyzer-max-auto-adjusts", None, "100", "10"),
         OptionStruct("int", "profile", "--run-config-search-min-concurrency", None, "2", "1"),
         OptionStruct("int", "profile", "--run-config-search-max-concurrency", None, "100", "1024"),
-        OptionStruct("int", "profile", "--run-config-search-min-request-rate-range", None, "2", "1"),
-        OptionStruct("int", "profile", "--run-config-search-max-request-rate-range", None, "100", "1024"),
+        OptionStruct("int", "profile", "--run-config-search-min-request-rate-range", None, "2", "1024"),
```

Comment:
I don't think this should say 'range' (here and many other places).
PA has options for concurrency-range and request-rate-range, but that allows them to set multiple concurrencies or multiple request rates to sweep over.  From our point of view it is just 'concurrency' and 'request-rate'.

Reply:
Removed range